contents/localization.html

```diff
@@ -0,0 +1,4 @@
+<h1>Localization</h1>
+<p>Firefox products are developed for a global audience. More than 70 percent of Firefox users live outside of North America. Our products are localized for more than 80 languages and regional dialects. 
+<p>Meeting the needs of our users around the world and working to inject more Firefox personality into copy will always be a balancing act. Copy with idioms, humor or complex sentence structures may pose challenges for localizers; your pithy English phrase may be misunderstood, or require 50 percent more copy to translate effectively. Worse, it may simply not be appropriate for a region’s sensibilities. The good news is that writing well for the web &mdash; short sentences, clear hierarchy, non-ambiguous language &mdash; will go far to support our localization efforts. <a ref="https://wiki.mozilla.org/L10n:WorldReady">Visit the localization team’s wiki</a> for more advice on the specifics of writing for localization. 
+<p>Localization is a partnership between writer, developer, and localizer. Be open to feedback from developers about copy structure. <a href="https://developer.mozilla.org/en- US/docs/Mozilla/Localization/Localization_content_best_practices" View these guidelines</a> for a bit of insight into how developers account for localization in code. 
```

Comment:
There are a couple of problems with the link in this line…
The `href` is `https://developer.mozilla.org/en- US/docs/Mozilla/Localization/Localization_content_best_practices`, but should be `https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices` (without the space).
And you need a `>` after the `"` to make sure you link the correct text.

(As a side note, if your editor can wrap the lines to 80 or 100 characters, it'll make commenting a little easier…)